db/migrate/20200525103646_move_characters_to_token_templates.rb

```diff
@@ -0,0 +1,73 @@
+class MoveCharactersToTokenTemplates < ActiveRecord::Migration[6.0]
+  def up
+    execute <<~SQL
+      INSERT INTO token_templates(
+        id,
+        type,
+        campaign_id,
+        name,
+        size,
+        created_at,
+        updated_at
+      )
+      SELECT
+        id,
```

Comment:
It's a good thing we used uuids!

Reply:
I'm wondering if there's ever an argument for anything else!